website/docs/docs/integrate/quickstart-semantic-layer.md

```diff
@@ -0,0 +1,292 @@
+---
+title: "Quickstart"
+id: quickstart-semantic-layer
+description: "Define metrics and set up the dbt Semantic Layer"
+sidebar_label: "Quickstart"
+---
+
+# dbt Semantic Layer quickstart
+
+:::info 📌
+
+The dbt Semantic Layer is currently available for public preview! With the dbt Semantic Layer, you'll be able to centrally define business metrics, remove code duplication and inconsistency, create self-service in downstream tools, and more! 
+
+
+For more info on public preview and plan availability, check out the [public preview](/docs/integrate/quickstart-semantic-layer#public-preview) and [product architecture](/docs/integrate/dbt-semantic-layer#product-architecture) section.
+
+:::
+
+
+## Public preview
+    
+We're excited to announce the dbt Semantic Layer is currently available for public preview, which means:
+
+&mdash; **Who?** The dbt Semantic Layer is open to all dbt Cloud tiers (Developer, Team, and Enterprise) during public preview. Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse dbt metrics in external tools, which requires access to the Metadata API. For more info on plan availability, check out our [product architecture](/docs/integrate/dbt-semantic-layer#product-architecture) section.
+
+&mdash; **What?** Public preview provides early access to new features, is supported and production ready, but not priced yet. Pricing for the dbt Semantic Layer will be introduced alongside the General Available (GA) release. There may also still be additions or modifications to product behavior. 
```

Comment:
Public Preview

Reply:
Suggest matching the language at [this doc](https://docs.google.com/document/d/1zy8R4NdBCp3TPWEQh0srXukqj45Zt7wZWOn1CAIEl6E/edit):

> Public Previews are meant to provide early access to new features. The Semantic Layer is stable and can be considered for production deployments, but there may still be some planned additions and modifications to product behaviors before moving to General Availability. We may also introduce new functionality that is not backwards compatible. Technical support is provided, and relevant service level objectives (SLOs) apply. Pricing for the dbt Semantic Layer will be introduced alongside the General Available (GA) release.